### PR TITLE
Add /getgamemode

### LINKED ISSUE #2113: Get a players gamemode easier
We could have a command that simply displays the players gamemode without the other spam of /seen. Simple command and it does not wrong to the server and I believe it could help. If we wanted to we could also color code the answers so if they are in Survival its green if its Creative red etc.

### PR DESCRIPTION
**Pull request by hitzzaf**
Fixes #2113 
Add's /getgamemode.
Aliases: /ggm, /getgm
Permissions: OPs
Description: "Get a players gamemode."
Usage </command> <playername>
Prints out as
"HitzzAF is in gamemode CREATIVE."

### CODE REVIEW

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_getgamemode.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        if (player == null) 
+        {
+            msg(ChatColor.GRAY + "Player not found!");
+        }
+        
+        msg(ChatColor.RED + player.getName() + " is in gamemode " + ChatColor.GREEN + player.getGameMode());
+
+        return true;
+    }
+
```
Comment (lemonsked) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_getgamemode.java:33]: useless whitespace.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_getgamemode.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package me.totalfreedom.totalfreedommod.command;
+
+import me.totalfreedom.totalfreedommod.rank.Rank;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = Rank.OP, source = SourceType.BOTH)
+@CommandParameters(description = "Get a players gamemode.", usage = "/<command> <playername>", aliases = "getgm,ggm")
```
Comment (lemonsked) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_getgamemode.java:10]: Put a space after the comma, please.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_getgamemode.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        if (args.length < 1) 
+        {
+            return false;
+        }
+        
+        final Player player = getPlayer(args[0]);
+        
+        if (player == null) 
+        {
+            msg(ChatColor.GRAY + "Player not found!");
```
Comment (lemonsked) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_getgamemode.java:26]: Use `msg(FreedomCommand.PLAYER_NOT_FOUND);`

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_getgamemode.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        }
```
Comment (lemonsked) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_getgamemode.java:27]: Missing return statement, code will continue running even though player is not found.

**Review by lemonsked — CHANGES_REQUESTED** [review]

**Comment by StevenLawson** [comment]
@hitzzaf Please [create separate branches for each feature](https://github.com/TotalFreedom/TotalFreedomMod/wiki/Contributing#quick-guide). This still has old commits for the /savebuild command that you proposed.

**Comment by Wild1145** [comment]
Thank you for your contribution. Sadly we have decided to close this issue as there has been no recent activity to this pull request. If you wish to continue working on this PR please re-open the issue and ensure that all requested changes are made.

The original issue has also been rejected as an issue.

**Pull request closed without merging**